build/kind/postgres.yaml

```diff
@@ -68,6 +68,10 @@ spec:
             - ssl_key_file=/var/lib/postgresql/tls/tls.key
             - -c
             - ssl_ca_file=/var/lib/postgresql/tls/ca.crt
+            - -c
+            - log_statement=all
+            - -c
+            - log_destination=stderr
```

Comment:
I could just check this myself I guess, but is this just a "echo all incoming requests" kind of thing?